ui/selectors/selectors.types.ts

```diff
@@ -1,6 +1,9 @@
 import type { InternalAccount } from '@metamask/keyring-api';
 import { SubjectMetadata } from '@metamask/permission-controller';
-import { KeyringType } from '../hooks/metamask-notifications/useProfileSyncing';
```

Comment:
I did not like this selector type importing the profile-sync type.

Instead copied the type here, and the accounts team can provide a better narrow type.